js/components/date.js

```diff
@@ -43,6 +47,14 @@ Fliplet.FormBuilder.field('date', {
   computed: {
     isWeb: function() {
       return Fliplet.Env.get('platform') === 'web';
+    },
+    isApplyCurrentDateFiled() {
```

Comment:
Install a spell checker https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

```suggestion
    applyCurrentDateField() {
```